src/client/src/models/Scene.js

```diff
@@ -234,12 +234,12 @@ const Scene = {
   },
 
   setFileName: function(fileName) {
-    var extensionCheck = fileName.split('.')
-    if (extensionCheck[extensionCheck.length - 1].toLowerCase() !== 'json') {
+    var extensionCheck = fileName.substr(fileName.length - 4).toLowerCase()
+    if (extensionCheck !== 'json') {
```

Comment:
It should be more readable to use a standard javascript library for validating with regex something like: `fileName.matches('.json$')` (idk if this is the syntax) and then using another standard library to trim the json when copying the file name.

Reply:
I'm pretty sure regex's are generally less readable than simple checks. It might not be to bad because it's a simple check. However that being said, it's a simple check. I added a comment to explain the section but I'm going to avoid putting a regex here, or anywhere I possibly can.